doc/ImplementingApplications/Steps2IntegrateElasticsearch/Steps2IntegrateElasticsearch.md

```diff
@@ -0,0 +1,255 @@
+## Integrate Elasticsearch (ES)
```

Comment:
I could see the file has not been included in ImplementingApplications.md file. We can add it to the following location : https://github.com/DanaSunal/ApplicationPattern/blob/docs/doc/ImplementingApplications/ImplementingApplications.md#individual-services .
Usually, the concepts part we include under "Concepts" and procedure we include under "Step-by-Step Guidelines". I feel this file contains both concepts and procedure, i would recommend splitting the file to ConceptOfElasticsearch.md  and Steps2IntegrateElasticsearch.md